src/elements/hx-file-tile/_shadow.scss

```diff
@@ -8,30 +8,25 @@
 
 // define Shadow DOM styles
 :host {
-  // reset buttons
-  button {
-    @include button-reset;
-  }
```

Comment:
revert this, always explicitly reset on `<button>` instead of each button variant